articles/quantum-techniques-6-PuttingItAllTogether.md

```diff
@@ -14,44 +15,159 @@ ms.topic: article
 ---
 
 # Putting it All Together: Teleportation #
+Let's return to the example of the teleportation circuit defined in [Quantum Circuits](quantum-concepts-8-QuantumCircuits.md). We're going to use this to illustrate the concepts we've learned so far. An explanation of quantum teleportation is provided below for those who are unfamiliar with the theory, followed by a walkthrough of the code implementation in Q#. 
 
-Let's return to the example of the teleportation circuit defined in [Quantum Circuits](quantum-concepts-8-QuantumCircuits.md). Shown below is a text-book quantum circuit that implements the teleportation, including the quantum part, the measurements, and the classically-controlled correction operations.
+## Quantum Teleportation: Theory
+Quantum teleportation is a technique for sending an unknown quantum state (which we'll refer to as the '__message__') from a qubit in one location to a qubit in another location (we'll refer to these qubits as '__here__' and '__there__', respectively). We can represent our __message__ as a vector using Dirac notation: 
+
+$$
+|\psi\rangle = \alpha|0\rangle + \beta|1\rangle
```

Comment:
I can't preview how this renders on the website (for some reason it allows me to preview only branches of this repo, not PRs from other repo), but I think ket states will look better the way we write them elsewhere, \ket{...}